portfolio/src/main/webapp/style.css

```diff
@@ -95,4 +95,15 @@ body {
      font-family: barrio;
      font-size: 24px;
      color: white;
+ }
+
+ #home-page-name {
+     position: absolute;
+     font-family: barrio;
+     writing-mode: vertical-rl;
+     text-orientation: upright;
+     color:white;
+     font-size: 72px;
```

Comment:
Lets group related things for easier readability and order consistency as you add more things here later 
so for now just group as font-family, font-size, color as in all the above element blocks

Reply:
sounds good ill change this in another branch!****